backend/mora/service/address_handler/phone.py

```diff
@@ -78,7 +78,7 @@ def _get_mo_properties(self):
     @classmethod
     def validate_value(cls, value):
         """Phone number is 8 digits, optionally with country code"""
-        if not re.match(r'\d{8}|\+45\d{8}', value):
+        if not re.match(r'^\d{8}$|^\+45\d{8}$', value):
```

Comment:
Erm, do we truly want to reject foreign phone numbers? (Probably not related to this issue, but still…)

(FWIW I think the City of Aarhus has an office in Brussels.)

Reply:
I'm just reimplementing the functionality we already have. We can add support for arbitrary country codes, but that seems out of scope of this.